grazer/src/main/java/edu/asu/diging/grazer/authentication/AuthenticationSuccessHandlerImpl.java

```diff
@@ -0,0 +1,22 @@
+package edu.asu.diging.grazer.authentication;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.SavedRequestAwareAuthenticationSuccessHandler;
+import org.springframework.stereotype.Service;
+
+@Service("authSuccessHandler")
+public class AuthenticationSuccessHandlerImpl extends SavedRequestAwareAuthenticationSuccessHandler {
+
+    @Override
+    public void onAuthenticationSuccess(HttpServletRequest request, HttpServletResponse response,
+            Authentication authentication) throws IOException, ServletException {
+        Object obj = authentication.getPrincipal();
```

Comment:
Where is the object obj used ? I don't think the line is required if we do not intend to make changes to the principal object